src/test/java/com/karankumar/bookproject/backend/utils/PredefinedShelfUtilsTest.java

```diff
@@ -121,4 +140,51 @@ void shouldGetAllBooksInChosenShelves() {
         Assertions.assertTrue(actualBooks.containsAll(expectedBooks));
     }
 
-}
+    @Test
+    void testAllPredefinedShelves() {
+        SoftAssertions softly = new SoftAssertions();
+
+        PREDEFINED_SHELVES.stream().forEach(shelfName -> {
+            String errorMsg = MessageFormat.format(ERROR_MSG, shelfName);
+            softly.assertThat(PredefinedShelfUtils.isPredefinedShelf(shelfName)).as(errorMsg).isTrue();
```

Comment:
In line with our style guide, please put method calls on separate lines when method chaining.